data/World/Water Temple MQ.json

```diff
@@ -62,9 +196,82 @@
                 Hover_Boots or can_use(Scarecrow) or logic_water_north_basement_ledge_jump",
             "Water Temple MQ GS Triple Wall Torch": "
                 can_use(Fire_Arrows) and (Hover_Boots or can_use(Scarecrow))",
+            "Water Temple MQ Triple Wall Torch Pot 1": "
+                can_use(Fire_Arrows) and (Hover_Boots or can_use(Scarecrow))",
+            "Water Temple MQ Triple Wall Torch Pot 2": "
+                can_use(Fire_Arrows) and (Hover_Boots or can_use(Scarecrow))",
+            "Water Temple MQ Triple Wall Torch Pot 3": "
+                can_use(Fire_Arrows) and (Hover_Boots or can_use(Scarecrow))",
+            "Water Temple MQ Triple Wall Torch Pot 4": "
+                can_use(Fire_Arrows) and (Hover_Boots or can_use(Scarecrow))",
             "Water Temple MQ GS Freestanding Key Area": "
                 logic_water_mq_locked_gs or ((Small_Key_Water_Temple, 2) and
-                    (Hover_Boots or can_use(Scarecrow) or logic_water_north_basement_ledge_jump))"
+                    (Hover_Boots or can_use(Scarecrow) or logic_water_north_basement_ledge_jump))",
+            "Water Temple MQ Dodongo Room Pot 1": "
+                logic_water_mq_locked_gs or ((Small_Key_Water_Temple, 2) and
+                    (Hover_Boots or can_use(Scarecrow) or logic_water_north_basement_ledge_jump))",
+            "Water Temple MQ Dodongo Room Pot 2": "
+                logic_water_mq_locked_gs or ((Small_Key_Water_Temple, 2) and
+                    (Hover_Boots or can_use(Scarecrow) or logic_water_north_basement_ledge_jump))",
+            "Water Temple MQ Freestanding Key Room Pot": "
+                Hover_Boots or can_use(Scarecrow) or logic_water_north_basement_ledge_jump",
```

Comment:
I have a PR in that fixes a minor issue with this room, and so this applies to your pot and crates as well, that if you don't have the north basement jump trick enabled, but for some reason do have the invisible hookshot target trick enabled, you can reverse through the dodongo room to reach the freestanding key room, if you have the second key.